cmd/apply.go

```diff
@@ -98,8 +109,23 @@ func apply(c *cobra.Command, _ []string) error {
 	return nil
 }
 
+func priorityTypeOf(prioirty string) (spanner.PriorityType, error) {
+	switch prioirty {
+	case "high":
+		return spanner.PriorityTypeHigh, nil
+	case "medium":
+		return spanner.PriorityTypeMedium, nil
+	case "low":
+		return spanner.PriorityTypeLow, nil
+	default:
+		return 0, fmt.Errorf("%s is unsupported priority, it must be one of high, medium, or low", priority)
```

Comment:
Given: https://github.com/cloudspannerecosystem/wrench/pull/37/files#r644431377, 
```suggestion
		return 0, fmt.Errorf("%s is unsupported priority, it must be one of %s, %s, or %s", priority, priorityHigh, priorityMedium, priorityLow)
```